action/lib/common.js

```diff
@@ -26,14 +26,21 @@ function triggerComponents(triggerName) {
     };
 }
 
+function addHTTPS(url) {
+    if (!/^(?:f|ht)tps?\:\/\//.test(url)) {
```

Comment:
i don't think `ftps` is relevant here - shouldn't this be only http or https?

Reply:
For sure. Will fix it =)